Landing page for the documentation (#1027)
climate-machine/CLIMA: docs/src/index.md

# CLIMA

```@meta
CurrentModule = CLIMA
```

`CLIMA` is a software package that models the evolution of the Earth system over weeks to centuries.
`CLIMA` solves three-dimensional partial differential equations for the distributions of water, momentum, energy, 
and tracers such as carbon in the atmosphere, oceans, and on land.

`CLIMA` will harness a wide range of Earth observations and data generated computationally to predict the evolution of Earth’s climate and features such as 
droughts, rainfall extremes, and high-impact storms.

## Subcomponents

> `CLIMA` couples three models for the subcomponents of the Earth system:

Reviewer: ```suggestion
`CLIMA` currently consists of three models for the subcomponents of the Earth system:
```

> * `CLIMA.Atmos`: A model of the fluid mechanics of the atmosphere and its interactaction with solar radiation and phase changes of water that occur, for example, in clouds.

Reviewer 2: 'interactation' => 'interaction'

* `CLIMA.Ocean`: A model for the fluid mechanics of the ocean and its distributions of heat, salinity, carbon, and other tracers.
* `CLIMA.Land`: A model for the flow of energy and water in soils and on the land surface, for the biophysics of vegetation on land, and for the transfer and storage of carbon in the land biosphere.

> The subcomponents are coupled by exchanging water, momentum, energy, and tracers such as carbon dioxide across their boundaries.

Reviewer: ```suggestion
The subcomponents will be coupled by exchanging water, momentum, energy, and tracers such as carbon dioxide across their boundaries.
```

# CLIMA

```@meta
CurrentModule = CLIMA
```

`CLIMA` is a software package that models the evolution of the Earth system over weeks to centuries.

> `CLIMA` solves partial differential equations that conserve the three-dimensional distributions of water, momentum, energy, 

Reviewer: ```suggestion
`CLIMA` solves partial differential equations for the three-dimensional distributions of water, momentum, energy, 
```

> and tracers such as carbon between the Earth's atmosphere, ocean, and land surface.

Reviewer: ```suggestion
and tracers such as carbon in the atmosphere, oceans, and on land.  
```

> `CLIMA`'s purpose is to harness as much data as possible to predict droughts, heat waves, 

Reviewer: ```suggestion
`CLIMA` will harness a wide range of Earth observations and data generated computationally to predict the evolution of Earth’s climate and features such as 
```

> rainfall extremes, and the evolution of Earth's climate.

Reviewer: ```suggestion
droughts, rainfall extremes, and high-impact storms.
```

## Subcomponents

`CLIMA` couples three models for the subcomponents of the Earth system:
 

> * `CLIMA.Atmos`: A model for the motion, transfer of solar radiation, and distribution of water, heat, and other tracers in the Earth's atmosphere.

Reviewer: ```suggestion
* `CLIMA.Atmos`: A model of the fluid mechanics of the atmosphere and how it interacts with radiative transfer and phase changes of water, for example, in clouds.
```

> * `CLIMA.Ocean`: A model for the motion and distribution of temperature, salinity, and other tracers in the Earth's ocean.

Reviewer: ```suggestion
* `CLIMA.Ocean`: A model for the fluid mechanics and thermodynamics of ocean waters motion and the distribution of tracers  in it.
```

Author: I suppose if we say that `Clima.Ocean` is a model for "fluid mechanics", then we are referring to the fact that it is a discrete model for the continuous system? The way I think about it, one could think as the equations we solve as a model for motion. However, it is true that we solve discrete equations, rather than the continuous ones that are more properly the "model of fluid motion".

Author: I guess maybe I think of the equations themselves as the "mechanics". But divorcing myself from this view, perhaps the equations are a model for mechanics? This is the idea?

Author: Ok, I decided to make this change to refer to the code as a model for "mechanics", rather than "motion".

> * `CLIMA.Land`: A model for the transfer of carbon, water, heat, and solar radiation in the Earth's terrestrial biosphere.

Reviewer: ```suggestion
* `CLIMA.Land`: A model for energy and water flows in soils and on the land surface, for the biophysics of vegetation on land, and for the transfer and storage of carbon in the land biosphere.
```

> The subcomponents are coupled by exchanging water, momentum, energy, and tracers across their boundaries.

Reviewer: ```suggestion
The subcomponents are coupled by exchanging water, momentum, energy, and tracers such as carbon dioxide across their boundaries.
```

## Dynamical core

> A dynamical core based on discontinuous Galerkin numerical methods is used to discretize the physical balances that underlie each of `CLIMA`'s subcomponents.

Reviewer: ```suggestion
A dynamical core based on discontinuous Galerkin numerical methods is used to discretize the physical conservation laws that underlie each of `CLIMA`'s subcomponents.
```

> ## Installation

Author: Sounds good.
